packages/fast-components-react-base/src/viewport-positioner/README.md

```diff
@@ -0,0 +1,53 @@
+## Viewport positioner
+
+The *viewport positioner* component is a container that positions itself relative to an anchor element while trying to take maximum advantage of the space available in the viewport.  For example it could enable a menu to open above or below a control depending on the size and scroll positon of the viewport.
+
+### Usage
+
+The *viewport positioner* and the anchor element it is attached to must both be children of the specified viewport element. There should also be no other scrollable elements in the DOM hierarchy between these elements and the viewport.  The *viewport positioner* element will have its position attribute programmatically set to "relative" and the component also writes in values for the top, right, bottom, left, transform-origin and tranform attributes. 
+
+If no "anchor" is specified the component will be disabled and not attempt to adjust layout. If no viewport is specified the component will assume the viewport is the root document element of the page.
+
+Authors can specify a positioning mode for each axis or leave it "uncontrolled" and use other means (i.e. their own css) to manage the positioner's layout on that axis.  The "flipOutward" positioning mode ensures that the positioner does not overlap with the anchor on that axis and butts up against the outer edges of the anchor element.  With the "flipInward" positioning mode set the positioner will overlap with the anchor on the axis specified but will only extend beyond the anchor on the side with the most available space.
```

Comment:
update flipOutward and flipInward to inset

Reply:
yep